### PR TITLE
Hotels: Fix status bar on gallery

### PR DESCRIPTION
**Pull request by tbergq**
Gallery and payments as screens instead of new modal

### CODE REVIEW

File: app/hotels/src/navigation/singleHotel/SingleHotelStack.js [hunk 1]
```diff
@@ -17,10 +17,40 @@ import GalleryGrid from '../../gallery/GalleryGrid';
...
+      },
+    },
+    GalleryGrid: {
+      screen: withMappedProps(GalleryGrid, AdditionalPropsInjector),
+    },
+    Payment: {
+      screen: withMappedProps(PaymentScreen),
+      navigationOptions: {
+        mode: 'modal',
+        headerTitle: (
```
Comment (RobinCsl) [inline app/hotels/src/navigation/singleHotel/SingleHotelStack.js]: Can we move this to the PaymentScreen component?

Comment (tbergq) [reply app/hotels/src/navigation/singleHotel/SingleHotelStack.js]: Probably yes

Comment (tbergq) [reply app/hotels/src/navigation/singleHotel/SingleHotelStack.js]: But we will get this error 😄 

image

Comment (tbergq) [reply app/hotels/src/navigation/singleHotel/SingleHotelStack.js]: @RobinCsl 🚀

**Pull request merged**